csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -1027,13 +1027,6 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
     const int64_t tma_m = getM(params_->mma_macro);
     const int64_t tma_n = getN(params_->mma_macro);
 
-    NVF_ERROR(
-        tma_n >= 64,
-        "Scheduler only supports 128B swizzle that requires N dimension of MMA ",
-        "macro to be >= 64, but received ",
-        tma_n,
-        ".");
```

Comment:
Do we have restrictions on the size of swizzle anymore? `tma_n` could potentially be any multiple of 8 from 8 to 256. We at least support `tma_n == 64 || tma_n == 32 || tma_n == 16`, but for example do we support `tma_n == 8` or `tma_n == 24`?

Reply:
My understanding is we'll support tma_n sizes of 8 or 24 but without any swizzling. Please correct me if I am wrong.

Reply:
For now, if the inner dimension is not a multiple of 32, we won't use any swizzling.

Reply:
But it is expected to compile. Sounds good.